### ISSUE TITLE
Implement Jenkins Integration

### ISSUE THREAD

**Issue by wylie** [ebsco/edna#184]
Actually implement what ever we found to be the best thing to implement in #181 

To be completed after #181
- [ ] #181

**Comment by gjjones** [comment]
This and [atomic](https://github.com/ebsco/edna/blob/develop/docs/atomic.md) sound like they'd benefit from revisiting our CSS analyzers/metrics. Jenkins would actually then be useful to us. We'd need to talk a lot more though about what we'd want out of atomic to understand what metrics we'd actually care about.

**Comment by wylie** [comment]
Yep, that's another area where we need a bunch of work. We should have a new issue for analyzers/metrics that are dependent on the [Break Free](https://github.com/ebsco/edna/milestones/Break%20Free!) milestone being complete.
